tools/templates/assets/css/samples.css

```diff
@@ -0,0 +1,198 @@
+/* General ********************************************************************/
+
+html, body {
+  height: 100%;
+}
+
+a:link, a:visited, a:active {
+  color: #4183C4;
+}
+
+#samples, #samples .ui.menu, #samples .ui.table {
+  font-size: 15px;
+}
+
+/* Getting Started ************************************************************/
+
+#getting-started {
+  margin-bottom: 70px;
+  font-weight: bold;
+  color:#000 !important;
+  padding-top: 12px;
+  padding-bottom: 14px;
+}
+
+#getting-started a:link,
+#getting-started a:visited,
+#getting-started a:active {
+  color: #000;
+  text-decoration: underline;
+}
+
+#getting-started h3 {
+  display: inline;
+  font-size: 1.18rem;
+  position: relative;
+  top: 3px;
+}
+
+/* Documentation **************************************************************/
+
+#content {
+  padding-left: 15rem;
+  padding-top: 0;
+  height: 100%;
+}
+
+#doc {
+  padding-left: 42px;
+  padding-top: 29px;
+}
+
+#default-app-text {
+  text-align: center;
+  font-size: 20px;
+  font-family: monospace;
+}
+
+.hljs {
+  padding: 0;
+}
+
+.ui.segment {
+  box-shadow: none;
+  border-radius: 0;
+  border-width: 1px 0;
+}
+
+.example h3 {
+  margin-bottom: 0.5rem;
+}
+
+.example .segment {
+  margin-top: 0.5rem;
+}
+
+.ui.table thead th {
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+ol.ui.list li {
+  padding-bottom: 19px;
+}
+
+.ui.mini.loader {
+  height: 0.8rem;
+  font-size: 1em;
+  top: -2px;
+}
+
+.ui.orange.labels {
+  padding: 20px 0 40px 0;
+  text-align: center;
+}
+
+.ui.orange.labels .ui.label {
+  text-transform: capitalize;
+}
+
+.sample-col {
+  border-left: 3px solid #000;
+  color: #F0EBD8;
+}
+
+.sample-col .ui.header {
+  color: #F0EBD8;
+}
+
+.ui.grid>.sample-col.column,
+.ui.grid>.sample-col.row,
+.ui.grid>.row>.sample-col.column {
+  background-color: #312F2F;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+/* Profile Page ***************************************************************/
+
+.profile {
+  text-align: center;
+}
+
+.profile .ui.table {
+  margin: 30px auto;
+}
+
+.profile .ui.table th {
+  text-align: center;
+}
+
+/* Sample App *****************************************************************/
+
+#okta-sign-in.main-container {
+  background-color: transparent;
+  border-width: 0;
+  color: #fff;
+  box-shadow: none;
+}
+
+#okta-sign-in .infobox,
+#okta-sign-in .infobox-md {
+  border: 0;
+  color: #000;
+  border-radius: 0;
+}
+
+#okta-sign-in .infobox:before,
+#okta-sign-in .infobox-md:before,
+#okta-sign-in .infobox-compact:before {
+  top: 0;
+  bottom: 0;
+  border-radius: 0;
+}
+
+#okta-sign-in.auth-container {
+  margin-top: 0;
+}
+
+#okta-sign-in .okta-sign-in-header {
+  border-width: 0;
+}
+
+#okta-sign-in h2 {
+  color: #F0EBD8;
+  font-weight: normal;
```

Comment:
Following up: I mistook this comment for another element - in this case, the element is defined in the Sign-In Widget, so at this level (the sample app), we don't have control over the html itself - we're just customizing the look for our app.